docs/3.0.0-beta.x/getting-started/introduction.md

```diff
@@ -15,7 +15,7 @@ It's the origin purpose of the project.
 
 ### Custom content structure
 
-With the admin panel of Strapi, You can generate the admin panel in just a few clicks, and get your whole CMS setup in a few minutes.
+With the admin panel of Strapi, you can generate the admin panel in just a few clicks, and get your whole CMS setup in a few minutes.
```

Comment:
I think this needs to be reworded.

You can generate the admin panel in a few clicks and get your whole CMS setup in a few minutes.

Reply:
Fixed